app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,10 @@
         <meta-data
             android:name="com.google.android.geo.API_KEY"
             android:value="AIzaSyBIXrmHklJ3kH7Cb8oOIPxTGGBCbt6Hh2o" />
+        <meta-data
+            android:name="preloaded_fonts"
+            android:resource="@array/preloaded_fonts" />
+
```

Comment:
I was messing around with some font stuff, any changes related to **fonts** in this PR is just me adding a test font to try it out and see how it looks. I can maybe try changing the fonts of the whole project later, if we have the time.